### PR TITLE
Add Trans to our warning banner - a working solution until refactor this component

### PR DESCRIPTION
**Pull request by khalidelaggan**
# Description

>This is a temporary workable solution until I refactor the component 

Fixes # (issue)

> Please delete options that are not relevant.

- [] New feature
- [Yes] Bug fix
- [ ] Technical debt
- [ ] This change requires a documentation update

# Checklist:

- [Yes ] My code follows the style guidelines of this project
- [Yes ] I have performed a self-review of my own code
- [] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made any needed changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

# Required followup work

> Will need to follow up with @amazingphilippe  the refactor work if needed.

### CODE REVIEW

File: f2/src/components/warning-banner/index.js [hunk 1]
```diff
@@ -2,6 +2,7 @@
 /** @jsx jsx **/
 import { jsx } from '@emotion/core'
 import PropTypes from 'prop-types'
+import { Trans } from '@lingui/macro'
 import { Alert, AlertIcon, AlertDescription } from '@chakra-ui/core'
 
 export const WarningBanner = props => {
```
Comment (amazingphilippe) [inline f2/src/components/warning-banner/index.js]: remove message from the props list

File: f2/src/components/warning-banner/index.js [hunk 2]
```diff
@@ -10,7 +11,7 @@ export const WarningBanner = props => {
   return (
     <Alert p={4} status={status} bg={`${bg}.400`} fontFamily="body">
       <AlertIcon mt={0} color="black" />
-      <AlertDescription color="black">{message}</AlertDescription>
+      <AlertDescription color="black"><Trans id="banner.warning" /></AlertDescription>
```
Comment (amazingphilippe) [inline f2/src/components/warning-banner/index.js]: Replace the Trans component with props.children. App.js already contains The exact same Trans tag as a child of the warning banner component!

then 🚢

Comment (amazingphilippe) [reply f2/src/components/warning-banner/index.js]: *also remove the import to prevent lint error

Comment (khalidelaggan) [reply f2/src/components/warning-banner/index.js]: Done - thank you for this 👍

File: f2/src/components/warning-banner/index.js (hunk 1 above)
Comment (sastels) [reply f2/src/components/warning-banner/index.js]: https://github.com/cds-snc/report-a-cybercrime/blob/713a5b80399400a41435b4209d45f1d6ee8aabd1/f2/src/components/warning-banner/index.js#L8

Comment (khalidelaggan) [reply f2/src/components/warning-banner/index.js]: oh no, it's fixed now :)

**Pull request merged**